modules/openapi-generator/src/main/resources/Java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -721,7 +722,7 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
 
         RequestEntity<Object> requestEntity = requestBuilder.body(selectBody(body, formParams, contentType));
 
-        ResponseEntity<T> responseEntity = restTemplate.exchange(requestEntity, returnType);
+        ResponseEntity<T> responseEntity = makeServiceCallWithRetry(requestEntity, returnType, 3, 10l);
```

Comment:
what about making maxAttempts, waitTimeMillis configurable as properties of ApiClient.java?